SPC Categorial Outlook Display.py

```diff
@@ -45,8 +45,8 @@ def display_spc_outlook(outlook_data):
     current_directory = os.path.dirname(os.path.abspath(__file__))
     states_shapefile = os.path.join(current_directory, 's_11au16.shp')  
     states = gpd.read_file(states_shapefile)  
-    states.plot(ax=ax, edgecolor='white', lw=0.5)
-    ax.set_facecolor("black")
+    states.plot(ax=ax, edgecolor='white', lw=0.5) #CHANGED: Remove facecolor (Added right below), and changed edgecolor to 'white' to contrast with the black 
+    ax.set_facecolor("black") #NEW: Background of the CONUS Shapefile will be Black

```

Comment:
All the Fixes and changed are right here.

# Fixes
- Fixed my attempt at having a black background by adding "ax.set_facecolor("black")" changing the facecolor (background) to black

# Changes
- Changed the edgecolor on the CONUS Shapefile to White to better contrast with the new background